requirements/constraints.txt

```diff
@@ -0,0 +1,20 @@
+# Version constraints for pip-installation.
+#
+# This file doesn't install any packages. It specifies version constraints
+# that will be applied if a package is needed.
+#
+# When pinning something here, please provide an explanation of why.  Ideally,
+# link to other information that will help people in the future to remove the
+# pin when possible.  Writing an issue against the offending project and
+# linking to it here is good.
+
+# TODO: Many pinned dependencies should be unpinned and/or moved to this constraints file.
+
+# Use latest Django LTS version
+Django<1.8.19
```

Comment:
can we move this up to Django 1.11 at least?

Reply:
If I upgrade it now to a newer version, the tests will not pass anymore. So we are planning to do this upgrade in a follow-up PR.